xbmc/pvr/windows/GUIWindowPVRRecordings.cpp

```diff
@@ -99,6 +100,19 @@ bool CGUIWindowPVRRecordingsBase::OnAction(const CAction& action)
       return true;
     }
   }
+  else if (action.GetID() == ACTION_TOGGLE_WATCHED)
+  {
+    CFileItemPtr pItem = m_vecItems->Get(m_viewControl.GetSelectedItem());
+    if (pItem->IsParentFolder())
+      return false;
+
+    if (pItem && (pItem->HasPVRRecordingInfoTag() || pItem->m_bIsFolder))
+    {
+      CVideoLibraryQueue::GetInstance().MarkAsWatched(pItem, pItem->GetOverlayImage() != "OverlayWatched.png");
```

Comment:
> The `ColumnLimit` in `.clang-format` is set to `100` which defines line length [...]

https://github.com/xbmc/xbmc/blob/master/docs/CODE_GUIDELINES.md#3-formatting

Reply:
No problems...used a variable instead to shorten line length